cameraViewEx/src/main/api21/com/priyankvasa/android/cameraviewex/Camera2.kt

```diff
@@ -945,7 +932,7 @@ internal open class Camera2(
                     CameraViewException("Af mode ${config.autoFocus.value} not supported by selected camera. Setting it to off."),
                     ErrorLevel.Warning
             )
-            config.autoFocus.value = Modes.DEFAULT_AUTO_FOCUS
+            config.autoFocus.postValue(Modes.DEFAULT_AUTO_FOCUS)
```

Comment:
Can you explain what is the issue and how this fix solves it? Usually `postValue()` is required to be used from a background task which is not the case here.

Reply:
I have tried again to reproduce this issue and was unable to! I will revert that change and submit a new PR.